### PR TITLE
Review website

### PR DESCRIPTION
**Pull request by ikus060**
Remove deadlink to wiki.
Add more recents articles.
Add reference to rdiffweb and Minarca.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h2>
...
 
-<p>Sol1 has officially taken over stewardship of rdiff-backup from February 2016. Sol1 has long been a contributor and user of rdiff-backup, and will maintain the open source nature of rdiff-backup, while bringing it into the modern era.</p>
+<p><a href="https://sol1.com.au/">Sol1</a> has officially taken over stewardship of rdiff-backup from February 2016. <a href="https://sol1.com.au/">Sol1</a> has long been a contributor and user of rdiff-backup, and will maintain the open source nature of rdiff-backup, while bringing it into the modern era.</p>
```
Comment (ottok) [inline index.html:38]: You mind if I also add a link to Seravo.com somewhere here?

Comment (ikus060) [reply index.html:38]: You could always add few lines about the contribution of seravo as being part of the new "team". I should probably do the same for my business. Me I don't mind. Having businesses behind an open source project is rewarding.

Comment (ericzolf) [reply index.html:38]: Fine with me as well.

Comment (davekempe) [reply index.html:38]: Maybe we can have a contributors past and present section? PS, you guys are doing an amazing job! great work!

**Pull request merged**